modules/cluster-logging-collector-syslog.adoc

```diff
@@ -32,6 +32,7 @@ Starting with the {product-title} 4.3, the process for using the *syslog* protoc
 You can forward logs to multiple syslog servers by specifying separate `<store>` stanzas in the configuration file.
 
 .Sample `syslog.conf`
+[source,terminal]
```

Comment:
Not sure what to do with this, but it's definitely not terminal output. It's a syslog configuration file.

I'd omit any highlighting for this, for now.